packages/lodestar/src/chain/chain.ts

```diff
@@ -124,7 +124,7 @@ export class BeaconChain extends (EventEmitter as {new (): ChainEventEmitter}) i
     return this.db.block.get(this.forkChoice.headBlockRoot());
   }
 
-  public async getBlockAtSlot(slot: Slot): Promise<SignedBeaconBlock | null> {
+  public async getCanonicalBlockAtSlot(slot: Slot): Promise<SignedBeaconBlock | null> {
```

Comment:
Could you add a JSDoc description to the function to clarify to future maintainers the meaning of CanonicalBlockAtSlot?

Reply:
hmm should be self explanatory, canonical is term for block in current head chain. What would you like be to write there?

Reply:
Explain that there can be more than 1 block per slot due to fork choice forks, and this function returns the block from the fork that we consider head. Basically Tuyen explained it great in his issue to be fixed with this PR.